warehouse/oidc/forms.py

```diff
@@ -20,26 +20,21 @@
 from warehouse.i18n import localize as _
 from warehouse.utils.project import PROJECT_NAME_RE
 
-_VALID_GITHUB_REPO = re.compile(r"^[a-zA-Z0-9-_.]+$")
-_VALID_GITHUB_OWNER = re.compile(r"^[a-zA-Z0-9][a-zA-Z0-9-]*$")
+_VALID_GITHUB_REPO = "[a-zA-Z0-9-_.]+"
+_VALID_GITHUB_OWNER = "[a-zA-Z0-9][a-zA-Z0-9-]*"
```

Comment:
Nitpick: use r-strings here (not strictly necessary, but IDEs will render them more nicely)

```suggestion
_VALID_GITHUB_REPO = r"[a-zA-Z0-9-_.]+"
_VALID_GITHUB_OWNER = r"[a-zA-Z0-9][a-zA-Z0-9-]*"
```